schemas/IDLs/org.gel.models.metrics.avro/1.0.1/ReportedVsGenetic.avdl

```diff
@@ -0,0 +1,153 @@
+@namespace("org.gel.models.report.avro")
+/**
+This protocol defines the metrics we calculate and store for genetic vs reported checks
+*/
+protocol ReportedVsGenetic {
+
+/** Definitions
+    TODO: Check if we want to have different karyotype definitions for XO clearcut/doubtful
+    */
+    enum Sex {UNKNOWN, MALE, FEMALE, OTHER}
+    enum KaryotypicSex {UNKNOWN, XX, XY, XO, XXY, XXX, XXYY, XXXY, XXXX, XYY, OTHER}
+    enum Query {yes, no, unknown}
+
+/**
+ReportedVsGeneticSummary
+========================================
+General information about the checks, versions, tools, and number of markers
+*/
+
+    record SamplesInfo {
+        string participantId;
+        string sampleId;
+        string deliveryId;
+    }
+
+    record ReportedVsGeneticSummary {
+        string RvsGCheckVersion;
+        string runDate;
+        string pathToDirectory;
+        union {null, double} numberOfMarkers;
+        union {null, double} numberOfMarkersLE;
+        string mendelErrorsTool;
+        string relatednessTool;
+        array<SamplesInfo> samplesInfo;
+    }
+
+/**
+Coverage-based sex
+========================================
+General information about the checks, versions, tools, and number of markers
+TODO: Consider removing Inferred karyotype
+*/
+    record CoverageBasedSex {
+        string sampleId;
+        union {null, KaryotypicSex} inferredKaryotype;
+        union {null, double} ratioChrX;
+        union {null, double} ratioChrY;
+        union {null, double} avgCnvChrX; // should be here?
+        union {null, double} avgCnvChrY; // should be here?
+    }
+
+/**
+Mendelian inconsistencies
+========================================
+- fmendel reports a line per nuclear family
+- imendel reports a line per member of the family
+*/
+    record PerFamilyMendelErrors {
+        string fatherId;
+        string motherId;
+        union {null, double} numberOfOffspring;
+        union {null, double} numberOfMendelErrors;
+    }
+
+    record IndividualMendelErrors {
+        string sampleId;
+        union {null, double} numberOfMendelErrors;
+        union {null, double} rateMendelErrors;
+    }
+
+    record AggregatedIndividualMendelErrors {
+        string sampleId;
+        union {null, double} totalnumberOfMendelErrors;
+    }
+
+    record TotalNumberOfMendelErrors {
+        union {null, double} family;
```

Comment:
`family` is the total of mendelian errors across all members of the family? We may want to change the name to something more concrete and include here a `familyId`?